tools/prowgen/pkg/testdata/simple.yaml

```diff
@@ -44,9 +44,20 @@ jobs:
     resources: custom
     command: [prow/command.sh]
     repos: [istio/istio]
-    modifier: [presubmit_skipped]
     trigger: "/test basic"
 
+  - name: multi-arch
+    types: [presubmit]
+    architectures: [amd64, arm64]
+    command: [prow/command.sh]
+    image: test
+
+  - name: multi-arch-param
+    types: [presubmit]
+    architectures: [amd64, arm64]
+    command: [prow/command.sh, $(params.arch)]
+    image: test
+
```

Comment:
Should we have an arm64 only job?

Reply:
This means "make an arm job and make an amd job". So it's mostly syntax sugar - they are independent